src/table/components/TableBodyWrapper.jsx

```diff
@@ -24,14 +24,15 @@ function TableBodyWrapper({
   const columnsStylingInfoJSON = JSON.stringify(columns.map((column) => column.stylingInfo));
   const setFocusedCellCoord = useContextSelector(TableContext, (value) => value.setFocusedCellCoord);
   const selectionDispatch = useContextSelector(TableContext, (value) => value.selectionDispatch);
-  // constraints.active: true - turn off interactions that affect the state of the visual
-  // representation including selection, zoom, scroll, etc.
-  // constraints.select: true - turn off selections.
-  const selectionsEnabled = !!selectionsAPI && !constraints.active && !constraints.select;
+  // active: turn off interactions that affect the state of the visual representation including selection, zoom, scroll, etc.
```

Comment:
Did you change those comments on purpose?

Reply:
no must be a merge error, good find